app/views/idv/doc_auth/_combined_upload.html.erb

```diff
@@ -0,0 +1,76 @@
+<% title t('titles.doc_auth.upload') %>
+
+<%= render 'idv/doc_auth/error_messages', flow_session: flow_session %>
+
+<%= render PageHeadingComponent.new do %>
+    <%= t('doc_auth.headings.combined_upload') %>
+<% end %>
+
+<p>
+    <%= t('doc_auth.info.combined_upload') %>
+</p>
+
+<hr class="margin-y-4" />
+
+<div class="grid-row">
+    <div class="grid-col-12 tablet:grid-col-3">
```

Comment:
Related to prior comment about image sizes, I might also suggest using auto / fill vs. hard-coded widths for the tablet arrangement, in case it starts looking off with the image sizes.

```suggestion
<div class="grid-row tablet:grid-gap">
    <div class="grid-col-12 tablet:grid-col-auto">
```

Below:

```html
    <div class="grid-col-12 tablet:grid-col-fill">
```

Reply:
Done in https://github.com/18F/identity-idp/pull/7849/commits/fc842599e35ad51914bfebdb124e977637a717db (note I also added a utility margin on the icons)